bin/dns/tests.js

```diff
@@ -49,54 +54,56 @@ ava.serial('dns zone export', async t => {
 const recordTypes = {
     a: {
         values: ['127.0.0.2', '127.0.0.3'],
-        lookup: ['127.0.0.2'],
+        expected: ['127.0.0.2'],
     },
     aaaa: {
         values: ['fdda:5cc1:23:4::1f', 'fdda:5cc1:23:4::1e'],
-        lookup: ['fdda:5cc1:23:4::1f'],
+        expected: ['fdda:5cc1:23:4::1f'],
     },
     cname: {
         values: ['cname.example.com.', 'cname2.example.com.'],
-        lookup: ['cname.example.com'],
+        expected: ['cname.example.com'],
     },
     txt: {
         values: ['"txt.example.com"', '"txt2.example.com"'],
-        lookup: [['"txt.example.com"']],
+        expected: [
+            [Buffer.from('"txt.example.com"')],
+        ],
     },
     mx: {
         values: ['10 mx.example.com.', '5 mx.example.com.'],
-        lookup: [
-            { priority: 10, exchange: 'mx.example.com' },
-        ],
+        expected: [{ preference: 10, exchange: 'mx.example.com' }],
+    },
+    caa: {
+        values: ['0 issue "comodoca.com"'],
+        expected: [{
+            flags: 0,
+            tag: 'issue',
+            value: '"comodoca.com"',
```

Comment:
@fredericosilva , we have problem there. Value ```0 issue "letsencrypt.org"```` is transformed to ```0 issue ""letsencrypt.org""``` in response. Notice double quota. 

I don't know if something inspire you to changes that in this way, but CloudFlare in API have quotation mark in API, but omit in UI:
![image](https://user-images.githubusercontent.com/3618479/64656148-c75f3380-d42e-11e9-8134-72b1c722225c.png)

Azure split each field in property. 
Google Cloud in UI & API require quotation marks.
AWS too:
![image](https://user-images.githubusercontent.com/3618479/64656377-8c113480-d42f-11e9-8dc7-941903d29477.png)